pkg/cli/mirror/manifests.go

```diff
@@ -255,6 +255,8 @@ func WriteICSPs(dir string, icsps []operatorv1alpha1.ImageContentSourcePolicy) e
 		return nil
 	}
 
+	klog.Infof("Writing ICSP manifests to %s", dir)
```

Comment:
Do we need to set log level for all logging here or we consider this essential log that should be always present?

Reply:
I would say essential since the user needs to be able to find and use those generated manifests.